hr_timesheet_portal/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2021 Hunki Enterprises BV
+# Copyright 2021 Sunflower IT
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Timesheet portal (editable)",
+    "summary": "Fill in timesheets via the portal",
+    "version": "13.0.1.0.0",
+    "development_status": "Alpha",
+    "category": "Website",
+    "website": "https://github.com/OCA/hr-timesheet",
```

Comment:
Thanks for this addition. Please attend to:
-    "website": "https://github.com/OCA/hr-timesheet",
+    "website": "https://github.com/OCA/timesheet",

Then tests should pass. ✨

Reply:
@KKamaa Can you attend to this?